### PR TITLE
more related efforts added

### PR DESCRIPTION
**Pull request by nikosft**

### CODE REVIEW

**Comment by aj-stein** [comment]
@nikosft sorry I missed this. I will review shortly and likely merge in. :-)

**Comment by aj-stein-nist** [comment]
@nikosft Sorry I missed this. I will review this shortly and likely merge it in. :-)

File: related-efforts.md [hunk 1]
```diff
@@ -11,22 +11,73 @@ has_toc: false
...
 
 ### Bill of Materials
 
 - [CycloneDX](https://cyclonedx.org)
 - [SPDX](https://spdx.dev/)
 - [Software Identificatino Tags - SWID (ISO/IEC 19770-2:2015)](https://www.iso.org/standard/65666.html)
 - [CoSWID - Concise Software Identification Tags (RFC9393)](https://datatracker.ietf.org/doc/rfc9393/)
+- [Microsoft's SBOM tool](https://github.com/microsoft/sbom-tool)
 
-### Digital Signatures
```
Comment (aj-stein-nist) [inline related-efforts.md:23]: Is there a reason we are removing OpenPubKey?

Comment (nikosft) [reply related-efforts.md:23]: It is not removed. I created a sub section "Related tools" and inside there another sub-section "Digital Signatures and Key Transparency" and I have moved it there together with some other tools.

Comment (aj-stein-nist) [reply related-efforts.md:23]: Oh, woops. Sorry. That makes a ton of sense.

**Review by aj-stein-nist — APPROVED** [review]

**Pull request merged**